onebuild/actions/predefined_actions.py

```diff
@@ -0,0 +1,5 @@
+class PredefinedActions:
+    VERSION = "ONEBUILD_VERSION"
+    HELP = "ONEBUILD_HELP"
+    LIST = "ONEBUILD_LIST"
+    INIT = "ONEBUILD_INIT"
```

Comment:
Let's avoid nested packaging structure - It becomes hard in distributing package & maintaining it.

Let's keep all files at `onebuild` package/module level.

WDYT @Ritabrata-TW ?

Reply:
Im not sure what you mean by nested packaging structure. Can you please explain ?

Reply:
action module inside onebuild

or 

command module inside onebuild

Reply:
Keeping everything inside the 1build module would make it quite clumsy in my opinion. Imagine moving all the commands to the base package. 

What do you think ?

Reply:
Yes, I agree it is going to be a big package - with nested modules - packaging becomes hard in my opinion 

Uploading package to pip & downloading might be an issue on some machines.

That's why I am proposing to keep one module `onebuild` and all the files inside it.

Reply:
okay. lets give it a try this way :)

Reply:
I will push the changes asap.